### PR TITLE
Updated utils.py & deleted old files

### PR DESCRIPTION
**Pull request by mjjeon**

### CODE REVIEW

File: appyters/Bulk_RNA_seq/utils.py [hunk 1]
```diff
@@ -43,6 +54,8 @@
 import scipy.stats as ss
 from sklearn.decomposition import PCA
 from sklearn.preprocessing import quantile_transform
```
Comment (u8sand) [inline appyters/Bulk_RNA_seq/utils.py:56]: Quantile Transform is **not the same as** Quantile Normalization as stated in the input form of your app.

```python
import pandas as pd
from sklearn.datasets import make_regression
from sklearn.preprocessing import quantile_transform
from maayanlab_bioinformatics.normalization.quantile import quantile_normalize
from matplotlib import pyplot as plt

X, _ = make_regression(n_samples=100, n_features=5)
X_quantile_trans_uniform = quantile_transform(X)
X_quantile_trans_norm = quantile_transform(X, output_distribution='normal')
X_quantile_norm = quantile_normalize(X)

fig, (ax1, ax2, ax3, ax4) = plt.subplots(ncols=4)
ax1.set_title('Input Data')
ax1.hist(X, label=[str(n) for n in range(X.shape[1])])
ax2.set_title('Quantile Transform (default = uniform)')
ax2.hist(X_quantile_trans_uniform, label=[str(n) for n in range(X.shape[1])])
ax3.set_title('Quantile Transform (normal)')
ax3.hist(X_quantile_trans_norm, label=[str(n) for n in range(X.shape[1])])
ax4.set_title('Quantile Normalize')
ax4.hist(X_quantile_norm, label=[str(n) for n in range(X.shape[1])])
plt.show()
```

image
